docker/akamai/Pipfile

```diff
@@ -0,0 +1,18 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+edgegrid-python = "*"
+requests = "*"
+olefile = "*"
+python-dateutil = "*"
+dateparser = "*"
+pyyaml = "*"
+funcy = "*"
```

Comment:
No need for all the dependencies that are already included in the base image. You should add only `edgegrid-python`

Reply:
forgot to change it when I change the base
Thanks